### ISSUE TITLE
Error if entrypoint name is 2item kebab case

### ISSUE THREAD

**Issue by 1natsu172** [wxt-dev/wxt#948]
### Describe the bug

I'm getting an error, reporting it. Not sure if it's a WXT issue or rollup issue, but it looks like the kebab-case name is passing to rollup and causing the error.

I'm surprised, because single kebab-case is not a problem.

| NG | OK |
|--------|--------|
| `ui-main-world.content` | `ui-main.content` | 

```
[17:56:08]  ERROR  Given name "_ui_main-world" is not a legal JS identifier. If you need this, you can try "output.extend: true".

  at getRollupError (/Users/1natsu/ghq/github.com/1natsu172/wxt/node_modules/.pnpm/rollup@4.19.0/node_modules/rollup/dist/es/shared/parseAst.js:392:41)
  at error (/Users/1natsu/ghq/github.com/1natsu172/wxt/node_modules/.pnpm/rollup@4.19.0/node_modules/rollup/dist/es/shared/parseAst.js:388:42)
  at Object.iife (/Users/1natsu/ghq/github.com/1natsu172/wxt/node_modules/.pnpm/rollup@4.19.0/node_modules/rollup/dist/es/shared/node-entry.js:15215:16)
  at Chunk.render (/Users/1natsu/ghq/github.com/1natsu172/wxt/node_modules/.pnpm/rollup@4.19.0/node_modules/rollup/dist/es/shared/node-entry.js:16292:27)
  at async Promise.all (index 0)
  at renderChunks (/Users/1natsu/ghq/github.com/1natsu172/wxt/node_modules/.pnpm/rollup@4.19.0/node_modules/rollup/dist/es/shared/node-entry.js:17941:28)
  at Bundle.generate (/Users/1natsu/ghq/github.com/1natsu172/wxt/node_modules/.pnpm/rollup@4.19.0/node_modules/rollup/dist/es/shared/node-entry.js:18179:13)
  at /Users/1natsu/ghq/github.com/1natsu172/wxt/node_modules/.pnpm/rollup@4.19.0/node_modules/rollup/dist/es/shared/node-entry.js:20726:27
  at catchUnfinishedHookActions (/Users/1natsu/ghq/github.com/1natsu172/wxt/node_modules/.pnpm/rollup@4.19.0/node_modules/rollup/dist/es/shared/node-entry.js:20153:16)
  at Module.build (/Users/1natsu/ghq/github.com/1natsu172/wxt/node_modules/.pnpm/vite@5.3.5_@types+node@20.14.12_sass@1.77.8/node_modules/vite/dist/node/chunks/dep-mCdpKltl.js:65125:16)

[17:56:08] ✖ Command failed after 2.251 s

[17:56:08]  ERROR  Failed to build ui-main-world

  at buildEntrypoints (/Users/1natsu/ghq/github.com/1natsu172/wxt/packages/wxt/dist/core/utils/building/build-entrypoints.mjs:19:13)
  at rebuild (/Users/1natsu/ghq/github.com/1natsu172/wxt/packages/wxt/dist/core/utils/building/rebuild.mjs:15:21)
  at internalBuild (/Users/1natsu/ghq/github.com/1natsu172/wxt/packages/wxt/dist/core/utils/building/internal-build.mjs:43:32)
  at buildAndOpenBrowser (/Users/1natsu/ghq/github.com/1natsu172/wxt/packages/wxt/dist/core/create-server.mjs:77:28)
  at Object.start (/Users/1natsu/ghq/github.com/1natsu172/wxt/packages/wxt/dist/core/create-server.mjs:42:9)
  at /Users/1natsu/ghq/github.com/1natsu172/wxt/packages/wxt/dist/cli/commands.mjs:32:5
  at CAC.<anonymous> (/Users/1natsu/ghq/github.com/1natsu172/wxt/packages/wxt/dist/cli/cli-utils.mjs:16:22)
  at /Users/1natsu/ghq/github.com/1natsu172/wxt/packages/wxt/dist/cli/index.mjs:11:1

file:///Users/1natsu/ghq/github.com/1natsu172/wxt/node_modules/.pnpm/@aklinker1+buildc@1.1.1_typescript@5.5.4/node_modules/@aklinker1/buildc/dist/cli.mjs:271
    throw Error("Commaned exited with code " + status);
          ^

Error: Commaned exited with code 1
    at execCommand (file:///Users/1natsu/ghq/github.com/1natsu172/wxt/node_modules/.pnpm/@aklinker1+buildc@1.1.1_typescript@5.5.4/node_modules/@aklinker1/buildc/dist/cli.mjs:271:11)
    at file:///Users/1natsu/ghq/github.com/1natsu172/wxt/node_modules/.pnpm/@aklinker1+buildc@1.1.1_typescript@5.5.4/node_modules/@aklinker1/buildc/dist/cli.mjs:199:7
    at async withLock (file:///Users/1natsu/ghq/github.com/1natsu172/wxt/node_modules/.pnpm/@aklinker1+buildc@1.1.1_typescript@5.5.4/node_modules/@aklinker1/buildc/dist/cli.mjs:152:12)
    at async buildPackage (file:///Users/1natsu/ghq/github.com/1natsu172/wxt/node_modules/.pnpm/@aklinker1+buildc@1.1.1_typescript@5.5.4/node_modules/@aklinker1/buildc/dist/cli.mjs:188:3)

Node.js v22.7.0
 ELIFECYCLE  Command failed with exit code 1.

```

### Reproduction

In the `wxt/packages/wxt-demo`

### Steps to reproduce

1. change dir name `ui.content` to `ui-main-world.content`
2. run `pnpm dev`

### System Info

```shell
System:
    OS: macOS 14.6.1
    CPU: (8) arm64 Apple M1
    Memory: 132.97 MB / 8.00 GB
    Shell: 3.7.0 - /opt/homebrew/bin/fish
  Binaries:
    Node: 22.7.0 - ~/.asdf/installs/nodejs/22.7.0/bin/node
    npm: 10.8.2 - ~/.asdf/plugins/nodejs/shims/npm
    pnpm: 9.6.0 - ~/.asdf/installs/nodejs/22.7.0/bin/pnpm
    bun: 1.1.25 - ~/.asdf/shims/bun
    Watchman: 2024.04.22.00 - /opt/homebrew/bin/watchman
  Browsers:
    Brave Browser: 111.1.49.120
    Chrome: 128.0.6613.86
    Safari: 17.6
  npmPackages:
    wxt: workspace:* => 0.19.8
```

### Used Package Manager

pnpm

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/wxt-dev/wxt/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://wxt.dev/guide/installation.html).
- [X] Check that there isn't [already an issue](https://github.com/wxt-dev/wxt/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/wxt-dev/wxt/discussions) or join our [Discord Chat Server](https://discord.gg/ZFsZqGery9).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by aklinker1** [comment]
This is WXT. Looks like this function is failing if there are multiple dashes? 

https://github.com/wxt-dev/wxt/blob/main/packages%2Fwxt%2Fsrc%2Fcore%2Futils%2Fstrings.ts#L8-L14

**Comment by aklinker1** [comment]
Let's just add https://unjs.io/packages/scule

**Comment by aklinker1** [comment]
Oh haha 🤦

```diff
-  return '_' + kebabCaseAlphanumeric(str.trim()).replace('-', '_');
+  return '_' + kebabCaseAlphanumeric(str.trim()).replaceAll('-', '_');
```

**Comment by 1natsu172** [comment]
`replace` and `replaceAll` are traps! 😅  And scule looks like a useful library!

Thanks for fast fixed!

**Comment by aklinker1** [comment]
Released in [`v0.19.9`](https://github.com/wxt-dev/wxt/releases/tag/wxt-v0.19.9)
